platform-docs-migration/README.md

```diff
@@ -0,0 +1,64 @@
+---
+creation_date: 2024-02-01
+issues:
+- https://github.com/giantswarm/giantswarm/issues/29503
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+summary: Revamp our docs to be Platform Dev ready considering new Cluster API (CAPI) architecture. Instead of modifying existing doc content find a way to create those from a clean plate and avoid stale content.
```

Comment:
```suggestion
summary: Revamp our docs to be Platform Dev ready considering the new Cluster API (CAPI) architecture. Instead of modifying existing docs content find a way to create a new site from a clean plate and avoid stale content.
```